wsd/Storage.cpp

```diff
@@ -65,6 +65,12 @@ bool StorageBase::WopiEnabled;
 bool StorageBase::SSLAsScheme = true;
 bool StorageBase::SSLEnabled = false;
 Util::RegexListMatcher StorageBase::WopiHosts;
+std::map<std::string, std::string> StorageBase::aliasHosts;
+std::map<std::string, std::string> StorageBase::aliasPorts;
+std::set<std::string> StorageBase::realHosts;
+std::string StorageBase::firstHost;
+
+#if !defined(BUILDING_TESTS)
```

Comment:
I assume there are tests, somewhere. Please include them in the PR to make reviewing more efficient.

It's also undesirable to have such defines. Let's discuss offline to figure out how to avoid them.